External/SPMmodified/xASL/xASL_io_SaveNifti.m

```diff
@@ -55,13 +55,13 @@ function xASL_io_SaveNifti(pathOrigNifti, pathNewNifti, imNew, nBits, bGZip, cha
 [pathstr, name0, ext0] = fileparts(pathNewNifti);
 if isempty(pathstr)
 	% If a file-name only, then add the full current path to avoid ambiguity
-	pathNewNifti = fullfile(xASL_adm_UnixPath(pwd()),[name0 ext0]);
+	pathNewNifti = fullfile(pwd(),[name0 ext0]);

```

Comment:
I would also check how this behaves on Windows, when you have illegal characters (such as spaces) in the path. But I would generally advocate not using spaces in pathnames, as good practice.

Reply:
> But I would generally advocate not using spaces in pathnames, as good practice.

Agree, but I think we should support it :/

Reply:
Michael tested this, so Windows should be fine.

Yes - we should not have spaces in pahtnames, but good to be able to deal with users who do that :)